docs/se/reactivemessaging/introduction.adoc

```diff
@@ -121,7 +121,7 @@ implicit ack before callback is executed is necessary.
 
 === Connector
 Connector concept is a way for connecting <<Channel,channels>> to external sources.
-To make xref:connector.adoc[creation and usage of connectors]
+To make creation and usage of connectors
 as easy and versatile as possible, Helidon SE Messaging uses same API for connectors
 like xref:../../mp/reactivemessaging/introduction.adoc[MicroProfile Reactive Messaging] does.
 This allows connectors to be usable in both flavors of Helidon with one limitation which is
```

Comment:
to be used in both Heldion MP and SE, however it has to be able to work without CDI.